src/settings/Settings.tsx

```diff
@@ -372,8 +372,8 @@ export const SETTINGS: { [setting: string]: ISetting } = {
         controller: new ServerSupportUnstableFeatureController(
             "feature_jump_to_date",
             defaultWatchManager,
-            ["org.matrix.msc3030"],
-            undefined,
+            [["org.matrix.msc3030"], ["org.matrix.msc3030.stable"]],
```

Comment:
Add stable unstable version (`org.matrix.msc3030.stable`) for jump to date [before `v1.6` is fully supported on a homeserver](https://github.com/matrix-org/synapse/issues/15089).

`org.matrix.msc3030.stable` wasn't included in [MSC3030](https://github.com/matrix-org/matrix-spec-proposals/pull/3030) but it seems like pattern that should have been included like [MSC2285](https://github.com/matrix-org/matrix-spec-proposals/pull/2285) as an example.

Are we okay with moving forward with this?

Reply:
Related discussion https://github.com/matrix-org/synapse/pull/15283#discussion_r1140669410